repositories/rhel-9.json

```diff
@@ -0,0 +1,66 @@
+{
+    "aarch64": [
+        {
+            "name": "baseos",
+            "baseurl": "http://download-node-02.eng.bos.redhat.com/rhel-9/composes/RHEL-9/RHEL-9.0.0-20210228.7/compose/BaseOS/aarch64/os",
```

Comment:
This should probably be the CDN address, and then we override it in tests, as this URL will expire.

Reply:
is there already CDN address for el9?